src/components/BooksList.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import BookItem from './BookItem';
+
+const BooksList = (props) => {
```

Comment:
- [ ] Your BooksList component must render a list of books using the Book component, you are handling it and using it correctly but you have to actually render a book list with sample books for this project.